### ISSUE TITLE
[Newcomers] Guide on adding a new pooled benchmark (WP2)

### ISSUE THREAD

**Issue by jeandut** [owkin/FLamby#31]
1. Make sure WP1 is validated by someone from owkin
2. Once WP1 is validated you should write the following empty file: `loss.py, model.py, benchmark.py, metric.py`
3. `loss.py` should implement a class called `BaselineLoss` from `torch.nn.modules.loss._Loss` naming are important
4. `model.py` should implement a class called `Baseline` inheriting `from torch.nn.Module`, Baseline should be able to forward the X batch from a dataset.
5. `metric.y` should implement a function called `metric` to assess the performance of the allgorithm
6.  Everything listed below should be importable by using `from flamby.datasets.fed_my_dataset import ...`
7. `benchmark.py` should provide a code sufficient to run the performance and test on the test set with pooled=True meaning the script should begin and end by:
```python
from flamby.datasets.fed_my_dataset import FedMyDataset, Baseline, Baselineloss, metric, NUM_CLIENTS, BATCH_SIZE, LR
from flamby.utils import evaluate_model_on_test
# the written code should do something like this
train_dataset = FedMyDataset(train=True, pooled=True)
train_dataloader = DataLoader(train_dataset, batch_size=BATCH_SIZE, shuffle=True)
test_dataset = FedMyDataset(train=False, pooled=True)
test_dataloader = DataLoader(test_dataset, batch_size=BATCH_SIZE, shuffle=False)
m = Baseline()
l = BaselineLoss()
for e in epochs:
    for X, y in train_dataloader:
        y_pred = m(X)

        ...
evaluate_model_on_tests(m, [test_dataloader], metric, use_gpu=True)
```
8. Note that BATCH_SIZE, LR should be both defined in `common.py` and importable with `from flamby.datasets.fed_my_dataset import ...`
